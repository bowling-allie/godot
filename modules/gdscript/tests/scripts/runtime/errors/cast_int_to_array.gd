#debug-only: Run-time invalid cast checks are disabled in release

func test():
	var integer: Variant = 1
	@warning_ignore("unsafe_cast")
	print(integer as Array)
