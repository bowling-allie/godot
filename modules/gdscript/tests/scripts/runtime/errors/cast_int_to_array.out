GDTEST_RUNTIME_ERROR
>> SCRIPT ERROR at runtime/errors/cast_int_to_array.gd:6 on test(): Invalid cast: could not convert value to 'Array'.
